ssh/src/main/java/ch/cyberduck/core/sftp/openssh/OpenSSHIdentityAgentConfigurator.java

```diff
@@ -33,7 +33,7 @@ public OpenSSHIdentityAgentConfigurator(final OpenSshConfig configuration) {
     }
 
     public String getIdentityAgent(final String alias) {
-        return configuration.lookup(alias).getIdentityAgent();
+        return configuration.lookup(alias).getIdentityAgent().getAbbreviatedPath();
```

Comment:
This will again lead to the same problem if you return the abbreviated path here. Thus you should call `getAbsolute()` I guess.

Reply:
right. Looks like we also need to handle the null case